Src/Base/AMReX_Algorithm.H

```diff
@@ -252,7 +252,7 @@ int clz (T x) noexcept;
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
 int clz_generic (std::uint8_t x) noexcept
 {
-    constexpr int clz_lookup[16] = { 4, 3, 2, 2, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0 };
+    static constexpr int clz_lookup[16] = { 4, 3, 2, 2, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
For some reason, nvc++ won't compile it with `static`.

Reply:
Maybe you can file a bug report. I suspect it should pretty easy to write a small reproducer.

Since we need to wait for the inline bug to be fixed anyway, we probably don't need to worry about this issue for now. If we need to work around this for nvc++, we could use `if !defined(__NVHPC)`. If we don't use `static`, the compiler might waste stack space unnecessarily.